packages/ocamlfind/ocamlfind.1.9.5/opam

```diff
@@ -13,7 +13,7 @@ authors: "Gerd Stolpmann <gerd@gerd-stolpmann.de>"
 homepage: "http://projects.camlcity.org/projects/findlib.html"
 bug-reports: "https://github.com/ocaml/ocamlfind/issues"
 depends: [
-  "ocaml" {>= "4.02.0"}
+  "ocaml" {>= "4.02.0" & < "5~" }
```

Comment:
@kit-ty-kate @dra27 if I recall correctly, 1.9.5 was the one introducing compatibility with ocaml 5.0

Should the bound be "< 5.1" or should we stick with the stricter bound that we have here?

Reply:
1.9.6 is the one that introduced OCaml 5.0 compatibility (see https://github.com/ocaml/ocamlfind/pull/52)
The constraint added by this PR is correct but a more standard way to write it would be:
```suggestion
  "ocaml" {>= "4.02.0" & < "5.0"}
```

Reply:
Thanks!